client/state/login/selectors.js

```diff
@@ -195,7 +189,7 @@ export const isFormDisabled = ( state ) => {
  * @returns {?string}        Authentication account type (e.g. 'regular', 'passwordless' ...)
  */
 export const getAuthAccountType = ( state ) => {
-	return get( state, 'login.authAccountType', null );
+	return state.login.authAccountType ?? null;
```

Comment:
No need to fallback. The state is either `null` or the `action.data.type` field of a REST response.

Reply:
Done.